omnipaxos_core/src/sequence_paxos/follower.rs

```diff
@@ -148,16 +148,11 @@ where
         if self.internal_storage.get_promise() == acc.n
             && self.state == (Role::Follower, Phase::Accept)
         {
-            if acc.seq_num == ACCSYNC_SEQ_NUM {
+            if acc.seq_num.counter == PREDEFINED_LEADER_FIRST_ACCEPT {
```

Comment:
I guess this is always (0,1) so we can change `PREDEFINED_LEADER_FIRST_ACCEPT` to that.